src/CalcViewModel/Common/TraceLogger.cpp

```diff
@@ -379,9 +389,86 @@ namespace CalculatorApp
             return;
 
         LoggingFields fields{};
-        fields.AddGuid(L"SessionGuid", sessionGuid);
-        fields.AddString(L"Mode", NavCategory::GetFriendlyName(mode)->Data());
+        fields.AddGuid(SESSION_GUID, sessionGuid);
+        fields.AddString(CALC_MODE, NavCategory::GetFriendlyName(mode)->Data());
         fields.AddUInt64(PDT_PRIVACY_DATA_TAG, PDT_PRODUCT_AND_SERVICE_USAGE);
         LogLevel2Event(EVENT_NAME_INPUT_PASTED, fields);
     }
+
+    void TraceLogger::LogShowHideButtonClicked(bool isHide)
+    {
+        if (!GetTraceLoggingProviderEnabled())
+            return;
+
+        LoggingFields fields{};
+        fields.AddGuid(SESSION_GUID, sessionGuid);
+        fields.AddString(CALC_MODE, GRPAHING_MODE);
+        fields.AddBoolean(L"ButtonState", isHide);
```

Comment:
It would be clearer from the data view if this were called something like “IsButtonHidden”, true/false rather than “ButtonState”, true/false

Reply:
I updated it to isHideButton